java.js

```diff
@@ -13,3 +13,60 @@ Items.forEach((n) => {
     menu.classList.toggle('hide');
   });
 });
+const projectDetails = [{
+  name: 'Keeping track of hundreds  of components website',
+  name2: 'Multi Post Stories',
+  detail: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  detail2: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  image: './img/Snapshoot.png',
+  technologies: [
+    'HTML/CSS',
+    'Ruby on Rails',
+    'JavaScript',
+  ],
+  live: 'https://github.com/Uchenna-ibute/Portfolio',
+  source: 'https://github.com/Uchenna-ibute',
+},
+];
```

Comment:
- Hi @Uchenna-ibute , Great job implementing the popup modal. But I believe the aim of this project is to generate a unique popup for each project card. In this case, you have used just one detail for all project cards. 🌸 


 Can we ensure to have at least 6? Based on your template, there are about 7 project cards (and see project buttons). I think each of them should have its own unique or slight difference. What do you think? 

Maybe each card name can have `Professional art Printing data 1`, `Professional art Printing data 2`, `Professional art Printing data 3` and so on. That way, when you want to show an employer your skills, you can always modify the details of each card showing individual projects you have built in future. 🤗